test/suites/zombie-offchain/test-simple-template-offchain.ts

```diff
@@ -0,0 +1,79 @@
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import { getHeaderFromRelay } from "../../util/relayInterface.ts";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { signAndSendAndInclude } from "../../util/block.ts";
+describeSuite({
+    id: "ZOF01",
+    title: "Zombie Offchain Tests",
+    foundationMethods: "zombie",
+    testCases: function ({ it, context }) {
+        let relayApi: ApiPromise;
+        let container2000Api: ApiPromise;
+
+        beforeAll(async () => {
+            relayApi = context.polkadotJs("Tanssi-relay");
+            container2000Api = context.polkadotJs("Container2000");
+
+            const relayNetwork = relayApi.consts.system.version.specName.toString();
+            expect(relayNetwork, "Relay API incorrect").to.contain("dancelight");
+
+            const container2000Network = container2000Api.consts.system.version.specName.toString();
+            const paraId2000 = (await container2000Api.query.parachainInfo.parachainId()).toString();
+            expect(container2000Network, "Container2000 API incorrect").to.contain("container-chain-template");
+            expect(paraId2000, "Container2000 API incorrect").to.be.equal("2000");
+
+            // Test block numbers in relay are 0 yet
+            const header2000 = await getHeaderFromRelay(relayApi, 2000);
+
+            expect(header2000.number.toNumber()).to.be.equal(0);
+        }, 50000);
+
+        it({
+            id: "T01",
+            title: "Offchain events are not emitted for simple container chain when offchain pallet testing is not enabled",
+            test: async function () {
+                const blockNum = (await container2000Api.rpc.chain.getBlock()).block.header.number.toNumber();
+                expect(blockNum).to.be.greaterThan(0);
+                await context.waitBlock(10, "Container2000");
+
+                const events = await container2000Api.query.system.events();
+                const offchainWorkerEvents = events.filter((a) => {
+                    return a.event.method == "SimpleOffchainEvent";
+                });
+                expect(offchainWorkerEvents.length).to.be.equal(0);
+            },
+        });
+
+        it({
+            id: "T02",
+            title: "Offchain events are not emitted for simple container chain when offchain pallet testing is enabled",
+            test: async function () {
+                const keyring = new Keyring({ type: "sr25519" });
+                const alice = keyring.addFromUri("//Alice", { name: "Alice default" });
+
```

Comment:
I am thinking whether we should add this to our main existing zombie test, I think i might be overkill to just add one single suite for this. What do you think @tmpolaczyk ?

Reply:
if we want this to be a separate suite, we shoudl at least run it when we merge to master in the CI

Reply:
Either way is ok, we have a job to manually run all the zombienet tests in ci.

If the offchain worker prints a log, we can use the existing test "Check collator logs to ensure common errors are fixed" to ensure that that particular log is not printed.

Reply:
so @chexware whatever you prefer to be honest,

Reply:
@girazoki seems that the current action for manually  running zombienet tests doesn't recognise the new moonwall config when ran it [here](https://github.com/moondance-labs/tanssi/actions/runs/11683481284). I assume that should work fine after merging the branch?

Reply:
it is very likely that you need to checkout to the appropriate github commit, otherwise it is probably using master?

Reply:
where can I see the job you guys are referring to?

Reply:
alright the problem is clear, when you use the workflow dispatch from your branch, it only takes the action from that branch but not the codebase (I think). you can try to add as input the sha like we do with https://github.com/moondance-labs/tanssi/blob/f8b66f38f93dc8fb2a1bbe9d2e44d14d21cbd1ca/.github/workflows/prepare-binary.yml#L7

then check how the action goes into the sha you input